src/main/java/net/greatstart/config/RootConfig.java

```diff
@@ -0,0 +1,10 @@
+//package ua.petshop.config;
+//
+//import org.springframework.context.annotation.ComponentScan;
+//import org.springframework.context.annotation.Configuration;
+//
+//@Configuration
+//@ComponentScan(basePackages = "ua.petshop")
+//public class RootConfig {
+//
+//}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)